pkg/appimport/appimport_test.go

```diff
@@ -13,69 +13,37 @@ import (
 
 	"github.com/drud/ddev/pkg/testcommon"
 	"github.com/drud/ddev/pkg/util"
-	"github.com/drud/ddev/pkg/version"
-	"github.com/drud/drud-go/utils/dockerutil"
-	"github.com/drud/drud-go/utils/system"
 	homedir "github.com/mitchellh/go-homedir"
 	"github.com/stretchr/testify/assert"
 )
 
 const netName = "ddev_default"
 
 var (
-	testArchiveURL  = "https://github.com/drud/wordpress/releases/download/v0.1.0/db.tar.gz"
 	testArchivePath = path.Join(testcommon.CreateTmpDir("appimport"), "db.tar.gz")
-	composePath     = path.Join("testing", "db-compose.yaml")
 	cwd             string
 )
 
 func TestMain(m *testing.M) {
-	err := system.DownloadFile(testArchivePath, testArchiveURL)
+	testFile, err := os.Create(testArchivePath)
```

Comment:
Oh, this is just a `touch db.tar.gz` ? A lot of work for a touch :) If it's just a touch, maybe the failure messages could say "dummy test file"

If it's just a touch, and if not required by all tests, may not belong in TestMain.

Regardless, I'm obviously confused by this, so it could use some explanatory comments.